app/views/desks/index.html.erb

```diff
@@ -2,6 +2,6 @@
 <% @desks.each do |desk| %>
   <ul>
     <li><%= link_to desk.name, desk_path(desk) %></li>
-    <li><%= desk.occupied %></li>
+    <li <%= "id=desk_state_#{desk.id}" %>><%= desk.occupied %></li>
```

Comment:
You can take the id outside of ruby land: `<li id=<%="desk_state_#{desk.id}" %>>...</li>`

Reply:
Ah thanks, that looks a lot cleaner.